GVFS/GVFS/CommandLine/UpgradeVerb.cs

```diff
@@ -13,12 +13,12 @@ public class UpgradeVerb : GVFSVerb.ForNoEnlistment
     {
         private const string UpgradeVerbName = "upgrade";
         private ITracer tracer;
-        private ProductUpgrader upgrader;
+        private IProductUpgrader upgrader;
         private InstallerPreRunChecker prerunChecker;
         private ProcessLauncher processLauncher;
 
         public UpgradeVerb(

```

Comment:
Why is this class not using the `UpgradeOrchestrator`?  I thought the responsibility and purpose of the orchestrator is to handle what needs to be done for the upgrade but it seems this class is doing a lot of that work as well.  Can some of this code be moved there and see what common code we have?

Reply:
True. This is a cleanup that needs to be done, still there in my TODO list. It is not related to Nuget upgrader & this PR though.